app/views/handlers/question.py

```diff
@@ -54,6 +50,29 @@ def questionnaire_store_updater(self):
             self.rendered_block.get("question"),
         )
 
+    @cached_property
+    def rendered_block(self):
+        transformed_block = transform_variants(
+            self.block,
+            self._schema,
+            self._questionnaire_store.metadata,
+            self._questionnaire_store.answer_store,
+            self._questionnaire_store.list_store,
+            self._current_location,
+        )
+        question_page_title = transformed_block.get(
+            "page_title"
+        ) or self._get_safe_page_title(transformed_block["question"]["title"])
+
+        self._set_page_title(question_page_title)
```

Comment:
As this is in the question handler, `question_page_title` can just be `page_title`.
```suggestion
        page_title = transformed_block.get(
            "page_title"
        ) or self._get_safe_page_title(transformed_block["question"]["title"])

        self._set_page_title(page_title)
```